package.json

```diff
@@ -29,10 +29,11 @@
   },
   "dependencies": {
     "css-loader": "^2.1.0",
+    "hammerjs": "2.0.8",
     "html-loader": "^0.5.5",
-    "picasso-plugin-hammer": "^0.25.3",
-    "picasso-plugin-q": "^0.25.3",
-    "picasso.js": "^0.25.3",
+    "picasso-plugin-hammer": "2.2.7",
+    "picasso-plugin-q": "2.2.7",
+    "picasso.js": "2.2.7",
```

Comment:
Didn't work with the current version of picasso. Bumped to the latest, but might be risky moving 2 major versions at once?

Reply:
Also, should we pin them? i can do that in a different PR first for all the deps if we want that

Reply:
As long as we test it properly it should be fine to bump major versions

Reply:
manually tested. download image/pdf is broken, but it was before as well.
I've also pinned the rest of the deps, to the version that is in package-lock